website/src/docs/hotchocolate/filtering/object-filter.md

```diff
@@ -0,0 +1,127 @@
+### Object Filter
```

Comment:
Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.